galaxy_ng/app/dynaconf_hooks.py

```diff
@@ -738,3 +739,17 @@ def configure_dab_required_settings(settings: Dynaconf) -> Dict[str, Any]:
         if key.isupper() and settings.get(key, notset) is notset:
             data[key] = getattr(dynamic_settings, key)
     return data
+
+
+def configure_dab_rbac_settings(settings: Dynaconf) -> Dict[str, Any]:
+    data = {}
+    if not hasattr(settings, 'ANSIBLE_BASE_MANAGED_ROLE_REGISTRY'):
```

Comment:
I usually avoid the [usage of](https://hynek.me/articles/hasattr/) `hasattr`
```suggestion
    if settings.get('ANSIBLE_BASE_MANAGED_ROLE_REGISTRY', None) is None:
```

Reply:
That's in general a good point, but with a quick look, RBAC has many settings that default to False, but none I see default to None.